test/tests/restart/scalar-var/tests

```diff
@@ -0,0 +1,19 @@
+[Tests]
+  design = 'MooseVariableBase.md'
+  issues = '#17471'
+  [restarting]
+    requirement = 'The system shall be able to restart a scalar variable in the presence of other variables with the standard sequence of:'
+    [part1]
+      type = Exodiff
+      input = part1.i
+      exodiff = part1_out.e
+      detail = 'run initial input'
+    []
+    [part2]
+      type = Exodiff
+      input = part2.i
+      exodiff = part2_out.e
+      detail = 'run restart input'
```

Comment:
Do you need to specify `part1` as a prerequisite?

Reply:
You're right that it needs the `prereq`. It doesn't really matter anymore because we always run tests sequentially within the same `tests` spec, but it's still good to have in case a user specifically requested par2 with a regex or I believe we do still have a way to force parallel test execution within the same directory...I just forget what it is. So I pushed up `prereq`. You'll need to re-approve.